x/oracle/keeper/prophecy.go

```diff
@@ -225,5 +229,5 @@ func (k Keeper) GetProphecyInfoWithScopeGlobalSequence(ctx sdk.Context,
 		globalSequence++
 		result = append(result, &prophecyInfo)
 	}
-
+	return result
```

Comment:
Interesting that you didn't need this explicit return in the previous version, and you need it now.

Reply:
the loop changed from without any condition to a fixed times.

Reply:
@jprupp Golang supports named return. that's why
https://stackoverflow.com/questions/45239409/empty-return-in-func-with-return-value-in-golang